src/main/kotlin/com/atlassian/performance/tools/infrastructure/api/jira/flow/PreStartFlow.kt

```diff
@@ -0,0 +1,8 @@
+package com.atlassian.performance.tools.infrastructure.api.jira.flow
+
+
+interface PreStartFlow : StartedJiraFlow, InstalledJiraFlow {
```

Comment:
Each flow points to the future. It's enough to point to the next step. We can rely on the next step to point to its next step. We'll have the access via transitivity.
In short, we should extend only one interface here. I believe it's the `StartedJiraFlow`, which should be renamed to `PostStartFlow`.